api/src/Entity/HasRootContentNodeTrait.php

```diff
@@ -3,50 +3,47 @@
 namespace App\Entity;
 
 use ApiPlatform\Core\Annotation\ApiProperty;
+use App\Entity\ContentNode\ColumnLayout;
+use App\Serializer\Normalizer\RelatedCollectionLink;
 use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Serializer\Annotation\Groups;
 use Symfony\Component\Validator\Constraints as Assert;
 
-#[ORM\Entity]
-#[ORM\InheritanceType('JOINED')]
-#[ORM\DiscriminatorColumn(name: 'entityType', type: 'string')]
-abstract class AbstractContentNodeOwner extends BaseEntity {
+trait HasRootContentNodeTrait {
```

Comment:
Probably "Geschmackssache", but I would rather keep the naming. Even though the `rootContentNode` property is at the moment specifically a `ColumnLayout`, the role it plays in this context is to be a root content node and not to be a generic column layout content node.

`HasColumnLayoutTrait` could, purely from a name, also apply to any other `ColumnLayout` (because they can also "have" other column layouts).